planner/core/prepare_test.go

```diff
@@ -14,7 +14,13 @@
 package core_test
 
 import (
+	dto "github.com/prometheus/client_model/go"
```

Comment:
please move this package to the lower block. separate it from the golang standard packages.

Reply:
Thank you for the comment. I've updated the PR.